pubspec.lock

```diff
@@ -7,7 +7,7 @@ packages:
       name: async
       url: "https://pub.dartlang.org"
     source: hosted
-    version: "2.5.0-nullsafety.1"
+    version: "2.5.0-nullsafety.3"
```

Comment:
never push pubspec.lock changes unless you have changed pubspec.yaml. Now, retract changes in this specific file and then push to the same branch.

Reply:
done!

Reply:
I meant the entire pubspec.lock file, not this specific line.